event_product/README.rst

```diff
@@ -0,0 +1,119 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==============
+Event Products
+==============
+
+This module extends the functionality of events and products to support
+combining them and allow you to have a *catalog* of events that you organize.
+
+Those events in your catalog can have variants, and any event can be linked to
+any product as long as:
+
+* The product's *Is an event* checkbox is enabled.
+* The product's event type is empty, or matches the event's.
+
+Examples:
+
+* Wedding.
+    * Variants:
+        * Until 50 attendees.
+        * Until 100 attendees.
+        * In beach.
+        * In countryfield.
+* Congress.
+    * Variants:
+        * Until 50 attendees.
+        * Until 100 attendees.
+* Opening.
+    * Variants:
+        * 1 day.
+        * 2 days.
+        * 3 days.
+
+Usage
+=====
+
+To create your *event products*, you need to:
+
+* Go to *Sales > Products > Products > Create*.
+* Give it a name, such as *Wedding*.
+* Enable *Is an event*.
+* Choose event type in *Information > Type of Event* (or leave blank for
+  generic events).
+* Press *Save*.
+
+To create variants for that, you need to:
+
+* Edit previous product.
+* Go to *Variants* tab.
+* Press *Add an Item*.
+* Add your attribute and values as usual.
+
+To link an event to any of those variants, you need to:
+
+* Go to *Marketing > Events > Events > Create*.
+* Set name, start and end dates.
+* Choose the same *Type of Event* as when creating the product (unless you
+  left it blank, in which case you do not need this).
+* Choose one of your variants in *Product*.
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/199/8.0
+
+Known issues / Roadmap
+======================
+
+* If you install this module, you will install *sale* too. If you do not want
+  to sale events, but just to handle internal ones, this can be disturbing
```

Comment:
If you do not intend to sell events, but just handle internal ones, this can be annoying since the sale module requires complex configuration with accounting and external dependencies.